api/src/main/java/uk/gov/hmcts/payment/api/dto/RemissionRequest.java

```diff
@@ -42,9 +43,11 @@
     @Valid
     private FeeDto fee;
 
-    @NotNull
     private String siteId;
```

Comment:
rid of site id same telephony payment also check for DTO **usage** across application.